torch/distributed/elastic/agent/server/api.py

```diff
@@ -41,8 +41,8 @@
 
 @dataclass
 class WorkerSpec:
-    """
-    Contains blueprint information about a particular type of worker.
+    """Contain blueprint information about a particular type of worker.
```

Comment:
This does not sound right. Should this be:
```suggestion
    """Blueprint information about a particular type of worker.
```